Advanced-Recruit-Training/src/BatteryStateOfChargeService.cpp

```diff
@@ -8,6 +8,11 @@ namespace
 BatteryStateOfChargeService::BatteryStateOfChargeService(double initialStateOfChargePercent)
 : initialStateOfChargePercent_(initialStateOfChargePercent)
 {
+    current_ = 0;
+    ampHoursUsed_ = BATTERY_AMP_HOUR_CAPACITY - BATTERY_AMP_HOUR_CAPACITY*(initialStateOfChargePercent_/100);
+    time_.setHMS(0,0,0,0);
```

Comment:
Use initialization lists for initializing private variables.
`BatteryStateOfChargeService::BatteryStateOfChargeService(double initialStateOfChargePercent)`
`: initialStateOfChargePercent_(initialStateOfChargePercent)`
`: ampHoursUsed_BATTERY_AMP_HOUR_CAPACITY - BATTERY_AMP_HOUR_CAPACITY*(initialStateOfChargePercent_/100);`
`: etc...`